descriptors/ModuleDescriptor-template.json

```diff
@@ -422,112 +143,50 @@
-        },
-        {
-          "methods": ["GET"],
-          "pathPattern": "/orders-storage/sources/{id}",
-          "permissionsRequired": ["orders-storage.sources.item.get"]
-        },
-        {
-          "methods": ["PUT"],
-          "pathPattern": "/orders-storage/sources/{id}",
-          "permissionsRequired": ["orders-storage.sources.item.put"]
-        },
-        {
-          "methods": ["DELETE"],
-          "pathPattern": "/orders-storage/sources/{id}",
-          "permissionsRequired": ["orders-storage.sources.item.delete"]
-        }
-      ]
-    },
-    {
-      "id": "orders-storage.vendor_details",
+      "id": "orders-storage.reporting-codes",
       "version": "3.0",
       "handlers": [
         {
           "methods": ["GET"],
-          "pathPattern": "/orders-storage/vendor_details",
-          "permissionsRequired": ["orders-storage.vendor_details.collection.get"]
+          "pathPattern": "/orders-storage/reporting-codes",
+          "permissionsRequired": ["orders-storage.reporting-codes.collection.get"]
         },
         {
           "methods": ["POST"],
-          "pathPattern": "/orders-storage/vendor_details",
-          "permissionsRequired": ["orders-storage.vendor_details.item.post"]
+          "pathPattern": "/orders-storage/reporting-codes",
+          "permissionsRequired": ["orders-storage.reporting-codes.item.post"]
         },
         {
           "methods": ["GET"],
-          "pathPattern": "/orders-storage/vendor_details/{id}",
-          "permissionsRequired": ["orders-storage.vendor_details.item.get"]
+          "pathPattern": "/orders-storage/reporting-codes/{id}",
+          "permissionsRequired": ["orders-storage.reporting-codes.item.get"]
         },
         {
           "methods": ["PUT"],
-          "pathPattern": "/orders-storage/vendor_details/{id}",
-          "permissionsRequired": ["orders-storage.vendor_details.item.put"]
+          "pathPattern": "/orders-storage/reporting-codes/{id}",
+          "permissionsRequired": ["orders-storage.reporting-codes.item.put"]
         },
         {
           "methods": ["DELETE"],
-          "pathPattern": "/orders-storage/vendor_details/{id}",
-          "permissionsRequired": ["orders-storage.vendor_details.item.delete"]
+          "pathPattern": "/orders-storage/reporting-codes/{id}",
+          "permissionsRequired": ["orders-storage.reporting-codes.item.delete"]
         }
       ]
     },
     {
-      "id": "orders-storage.po_number",
+      "id": "orders-storage.po-number",
       "version": "2.0",
       "handlers": [
         {
           "methods": ["GET"],
-          "pathPattern": "/orders-storage/po_number",
-          "permissionsRequired": ["orders-storage.po_number.get"]
+          "pathPattern": "/orders-storage/po-number",
+          "permissionsRequired": ["orders-storage.po-number.get"]
         }
       ]
     },
     {
       "id": "orders-storage.po-line-number",
-      "version": "1.0",
+      "version": "2.0",
```

Comment:
keep at 1.0 for now... bump at release time.

Reply:
Set at 1.0